pkg/v1/tkg/client/validate.go

```diff
@@ -1351,10 +1351,6 @@ func (c *TkgClient) ConfigureAndValidateCNIType(cniType string) error {
 // DistributeMachineDeploymentWorkers distributes machine deployment for worker nodes
 func (c *TkgClient) DistributeMachineDeploymentWorkers(workerMachineCount int64, isProdConfig, isManagementCluster bool, infraProviderName string) ([]int, error) { // nolint:gocyclo
 	workerCounts := make([]int, 3)
-	if infraProviderName != AWSProviderName && infraProviderName != AzureProviderName {
```

Comment:
Question: Are we considering the `docker` provider while we are removing this condition

Reply:
ah I wasn't thinking about docker. Should I revert this change, and skip the vSphereProvder as well in his conditional?

Reply:
I would say that would be the safe thing to do for now.

Reply:
Will do

Reply:
Done